ftw/iframeblock/tests/test_iframeblock.py

```diff
@@ -68,3 +68,66 @@ def test_scrolling_always_set_to_no(self, browser):
             'no',
             browser.css('iframe.iframeblock').first.attrib['scrolling']
         )
+
+    @browsing
+    def test_alternative_url_can_be_passed_as_request_param(self, browser):
+        """
+        When integrating other websites in iframes and indexing those
```

Comment:
@maethu I've updated the readme with the PR description.
I've also rephrased / cleaned up the the introduction and removed the confusing part about deployment.